src/function/validate.js

```diff
@@ -0,0 +1,16 @@
+export function isUpdatedReferenceValue(prev, curr) {
+  const prevData = JSON.stringify(prev);
+  const currentData = JSON.stringify(curr);
+
+  if (prevData === currentData) {
+    return false;
+  }
+  return true;
```

Comment:
아래처럼 축약해서 표현하셔도 좋을거 같아요

```suggestion
  return prevData !== currentData
```

Reply:
앟.. 이부분을 완전 놓치고 있었네요 .. 감사합니다!

Reply:
8649f8602f5c2aa9be4ac9c4c074cab7b27e6ea5 
수정하였습니다 😀